components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderDAO.java

```diff
@@ -251,12 +271,24 @@ public boolean addServiceProvider(SAMLSSOServiceProviderDO serviceProviderDO) th
             }
             registry.put(path, resource);
             if (log.isDebugEnabled()) {
-                log.debug("Service Provider " + serviceProviderDO.getIssuer() + " is added successfully.");
+                if (StringUtils.isNotBlank(serviceProviderDO.getIssuerQualifier())) {
+                    log.debug("Service Provider " + serviceProviderDO.getIssuer() + " with issuer "
```

Comment:
Change Service Provider to SAML2 Service Provider

Reply:
Changed